easybuild/easyconfigs/g/GD/GD-2.52-foss-2016b-Perl-5.24.0.eb

```diff
@@ -0,0 +1,31 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+
+easyblock = 'PerlModule'
+
+name = 'GD'
+version = '2.52'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://search.cpan.org/~lds/%(name)s/'
+description = """GD.pm - Interface to Gd Graphics Library"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['http://cpan.metacpan.org/authors/id/L/LD/LDS/']
+sources = [SOURCE_TAR_GZ]
+
+checksums = ['646a4b5ba587763ed791bad4f2f79e95']
+
+dependencies = [
+    ('Perl', '5.24.0'),
+    ('libgd', '2.2.3'),
```

Comment:
the existing easyconfig for this depends on libpng 1.6.24, so including a dependency on libpng 1.6.26 below isn't going to work...

There's a libgd 2.2.4 release, maybe include an updated easyconfig for libgd, and use the latest libpng 1.6.27 both in libgd and here?